### DISCUSSION TITLE
Missing versions 1.3 and 1.4

### DISCUSSION THREAD

**Discussion by kangarko** [MCPHackers/RetroMCP-Java#89] [General]
Hey! I know there is probably a good reason but I was just curious if there are plans on including 1.3.2 and 1.4.7 in the program.

**Comment by PhoenixVX** [comment]
There isn't a good reason. It's simply because the two people who are actively working on RMCP and know how it works haven't put forth the time for these versions to work.

**Reply by kangarko** [reply]
I see. Well the old MCP works fine for them. I'd love to put one version into a Git repo and see code changes between versions but the problem is, mappings are heavily changed even the names (i.e. player > thePlayer) betweer yours and the old MCP.

**Reply by PhoenixVX** [reply]
For mappings which have a MCP release, we simply port those to the Tiny mapping format. For versions which never had mappings, we are forced to match mappings from versions close to it. In any case, what you really want is *intermediary*, which is consistent between versions. https://github.com/DuvetMC might have what you're looking for which is intermediary for almost every version of MC.

**Reply by kangarko** [reply]
Thank you!
